docs/source/es/_toctree.yml

```
@@ -20,58 +20,59 @@
   - local: model_sharing
     title: Compartir un modelo
   title: Tutoriales
-- sections:
-  - sections:
-    - local: create_a_model
-      title: Crea una arquitectura personalizada
-    - local: custom_models
-      title: Compartir modelos personalizados
-    - local: run_scripts
-      title: Entrenamiento con scripts
-    - local: sagemaker
-      title: Ejecutar el entrenamiento en Amazon SageMaker
-    - local: converting_tensorflow_models
-      title: Convertir checkpoints de TensorFlow
-    - local: serialization
-      title: Exportar a ONNX
-    title: Uso general
-  - sections:
-    - local: fast_tokenizers
-      title: Usa tokenizadores de 🤗 Tokenizers
-    - local: multilingual
-      title: Modelos multilingües para inferencia
-    - sections:
-      - local: tasks/question_answering
-        title: Respuesta a preguntas
-      - local: tasks/language_modeling
-        title: Modelado de lenguaje
-      - local: tasks/summarization
-        title: Generación de resúmenes
-      - local: tasks/multiple_choice
-        title: Selección múltiple
-      title: Guías de tareas
+- section:
```

This needs to be `sections` otherwise its not recognized :)

```suggestion
- sections:
```

---

Done, thanks!